src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -199,5 +229,48 @@ export class MetaCampaignComponent implements OnInit {
       campaign.summary,
       campaign.bannerUri,
     );
+    // As per https://angular.io/guide/security#bypass-security-apis constructing `SafeResourceUrl`s with these appends should be safe.
+    if (campaign.video && campaign.video.provider === 'youtube') {
+      this.videoEmbedUrl = this.sanitizer.bypassSecurityTrustResourceUrl(`https://www.youtube.com/embed/${campaign.video.key}`);
+    } else if (campaign.video && campaign.video.provider === 'vimeo') {
+      this.videoEmbedUrl = this.sanitizer.bypassSecurityTrustResourceUrl(`https://player.vimeo.com/video/${campaign.video.key}`);
+    }
+  }
+
+  /**
+   * Set query params to this.query object, if any are available.
+   */
+  private setQueryParams() {
+    if (window.location.search) {
```

Comment:
This is not SSR safe (possibly why the boolean param workaround was needed). We should get params from Angular-injected params instead. This will be more durable e.g. when we move to loading data earlier such that presence of `campaign` is not an indicator of whether we are server-rendering or not.